distribution/docs/src/main/resources/content/_metadataReference/associations-attributes-table.adoc

```diff
@@ -34,6 +34,6 @@
 resources.
 |List of Strings
 |A valid URI.
-|https://infocorp.org/wikia/reference
+|\https://infocorp.org/wikia/reference
```

Comment:
❓ Was this added slash intentional?

Reply:
So the backslash escapes the URL to prevent it from becoming a hyperlink in the documentation?

Reply:
yes